pkg/mtbroker/ingress/ingress_handler_test.go

```diff
@@ -183,16 +202,25 @@ func TestHandler_ServeHTTP(t *testing.T) {
 				request.Header.Add(cehttp.ContentType, event.ApplicationCloudEventsJSON)
 			}
 
+			for _, b := range tc.brokers {
+				// Write the channel address in the broker status annotation
+				if b.Status.Annotations == nil {
+					b.Status.Annotations = make(map[string]string, 1)
+				}
+				b.Status.Annotations["channelAddress"] = s.URL
+			}
+
+			// TODO: use broker lister from "knative.dev/pkg/reconciler/testing" once
```

Comment:
It should be there, but it's in testing/v1:
https://github.com/knative/eventing/blob/master/pkg/reconciler/testing/v1/listers.go#L131

Reply:
Done: ce09c15321cf61ecf45a0aa59cc023afd4e7cc8e